flight/targets/aq32/fw/Makefile

```diff
@@ -272,7 +272,6 @@ CSTANDARD = -std=gnu99
 # Flags for C and C++ (arm-elf-gcc/arm-elf-g++)
 
 CFLAGS += -g$(DEBUGF)
```

Comment:
fast-math is removed everywhere because I've kept finding dangerous things with fast-math, and now we want to use NANs to signal no-spin.